README.md

```diff
@@ -139,6 +139,96 @@ command groups. The `NullOrchestrator` disables commands related to managing ser
 orchestrator=NullOrchestrator()
 ```
 
+#### HelmOrchestrator
+
+The project also includes a [helm](https://helm.sh/docs/intro/quickstart/) orchestrator for deploying charts to [kubernetes](https://kubernetes.io/) clusters.
+
+Create a new `resources` directory as follows:
+
+```bash
+drwxrwxr-x  resources/
+.rw-rw-r-- ├──  settings.yml
+drwxrwxr-x └──  templates/
+drwxrwxr-x    ├──  baseline/
+drwxrwxr-x    │  └──  cli/
+drwxrwxr-x    │     └──  helm/
+drwxrwxr-x    │        ├──  values/
+.rw-rw-r--    │        │  └──   values.yml
+drwxrwxr-x    │        ├──  values-files/
+.rw-rw-r--    │        └──   mychart.tgz
+drwxrwxr-x    └──  configurable/
+drwxrwxr-x       └──  cli/
+drwxrwxr-x          └──  home/
+drwxrwxr-x             └──  .kube/
+.rw-------                └──  config  # Overwrite this with a cluster specific config file. ie `~/.kube/config`.
+```
+
+You can then define the orchestrator:
+
+```python
+from appcli.orchestrators import HelmOrchestrator
+orchestrator = HelmOrchestrator(
+    # Name of the project.
+    release_name="myapp",
+
+    # Chart archive path (relative to `conf/templates/cli/helm/`).
+    # [Optional] Default is `chart/`
+    chart_location="mychart.tgz",
+
+    # Chart source path (absolute).
+    # [Optional] Default is `None`.
+    dev_chart_location="path/to/mychart"
```

Comment:
I don't really see the value of including this in the config. It's only for DEV purposes, we shouldn't ever publish an app with a hard-coded path from a dev's workstation, so people will always need to remember to clean up the file.

Let's just use an env var instead:
```
MYAPP_DEV_MODE=true MYAPP_DEV_MODE_HELM_CHART_PATH=/path/to/mychart ./myapp.py
```

We also need to explain it better. I.e. precisly why you added it. During development it would be slow to require packaging up the chart for any change, etc. This mechanism allows developers to iterate on the chart without needing to blah blah blah